dev/com.ibm.ws.st.ui/src/com/ibm/ws/st/ui/internal/download/RepositoryInfoDialog.java

```diff
@@ -305,7 +306,20 @@ private void init() {
             nameText.setText(site.getName());
 
             if (url.toString().startsWith(FileUtil.FILE_URI)) { // local site
-                locationText.setText(site.getURL().toString().replace(FileUtil.FILE_URI, ""));
+                String path;
+                try {
+                    File file = new File(url.toURI());
+                    path = file.getPath();
+                } catch (URISyntaxException e) {
+                    // This should not happen
+                    Trace.logError("Could not create a file from the URL: " + url, e);
+                    path = url.getFile();
+                    if (System.getProperty("os.name").toLowerCase().contains("windows")) {
+                        // remove the leading '/'
+                        path = path.substring(1);
```

Comment:
As discussed, just need to add a check for whether it's starting with a '/'

Reply:
Thanks for catching that.  I have fixed it.